File: Java/CodingBat/squirrelPlay.java
// http://codingbat.com/prob/p141061

public boolean squirrelPlay(int temp, boolean isSummer) {
  boolean temperatureIsGood = (temp >= 60 && temp <= 90);
  boolean temperatureIsGoodForSummer = (temp >= 60 && temp <= 100);
  
  if (temperatureIsGood && !isSummer) {
    return true;
  }
  else if (isSummer && temperatureIsGoodForSummer) {
    return true;
  }
  else {
    return false;
  }
}

File: Python/CodingBat/squirrel_play.py
# http://codingbat.com/prob/p135815

def squirrel_play(temp, is_summer):
  if is_summer:
    if temp >= 60 and temp <= 100:
      return True
  else:
    if temp >= 60 and temp <= 90:
      return True
      
  return False
